cmd/utils/dev_test.go

```diff
@@ -435,3 +436,60 @@ func Test_SelectDevFromManifest(t *testing.T) {
 	}
 
 }
+
+func Test_AskYesNo(t *testing.T) {
+	tests := []struct {
+		name     string
+		def      YesNoDefault
+		answer   string
+		expected bool
+	}{
+		{
+			name:     "ignores-default-when-answer",
+			def:      YesNoDefault_No,
+			answer:   "y\n",
+			expected: true,
+		},
+		{
+			name:     "honors-default-when-no-answer",
+			def:      YesNoDefault_Yes,
+			answer:   "\n",
+			expected: true,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			// Create a temp file to mock stdin
+			tmpFile, err := ioutil.TempFile("", "yes_no_tests")
```

Comment:
Deepsource pipeline for the PR is reporting `io/ioutil`  https://github.com/okteto/okteto/pull/3192/checks?check_run_id=9041048721 

what about creating a temp dir (`t.TempDir`), and writting here a file for this instead of using `ioutil.TempFile`. Here is an example: https://github.com/okteto/okteto/blob/master/cmd/build/v1/build_test.go#L131 that way it will be cleanup when the test is finished

Reply:
thank you @teresaromero! TIL that `ioutil` has been deprecated and that `t.TempDir` exists 👍🏻